common/gazebo_node.h

```diff
@@ -0,0 +1,28 @@
+#pragma once
+// Gazebo includes
+#include <gazebo/gazebo_config.h>
+#include <gazebo/transport/transport.hh>
+#include <gazebo/msgs/msgs.hh>
+#include <gazebo/common/common.hh>
+
+// Gazebo's API has changed between major releases. These changes are
+// accounted for with #if..#endif blocks in this file.
+#if GAZEBO_MAJOR_VERSION < 6
+#include <gazebo/gazebo.hh>
+#else
+#include <gazebo/gazebo_client.hh>
+#endif
```

Comment:
Let's drop support for Gazebo < v6 here.

Reply:
Sorry, I missed this. Fixed in commit 6045d5e